package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "angular-http-auth",
-  "version": "1.2.3",
+  "version": "1.3.0",
```

Comment:
The thing that seem strange to me is that the version `1.2.3` was published/tagged already. Now we have this pull request. As you say, it should not touch the version - OK, but now we have a new code, new feature, new behavior, but the version `1.2.3` stays untouched and it does not match the released version any more. 

In Java/Maven we keep `x.x.x-SNAPSHOT` during development. Then comes the release, the version gets cleared from snapshot suffix. Then the version gets bumped to the new number, the snapshot gets attached again and we have clear situation on what is the version of each commit.

If I understand correctly, the NPM way is to keep the old version number up until next release, is that correct?